addons/mail/static/src/rtc/call_action_list.js

```diff
@@ -51,6 +69,21 @@ export class CallActionList extends Component {
         }
     }
 
+    onClickDownloadLogs() {
+        const data = window.JSON.stringify(Object.fromEntries(this.rtc.state.logs));
+        const blob = new window.Blob([data], { type: "application/json" });
+        const url = window.URL.createObjectURL(blob);
```

Comment:
There should be no need to prefix these with `window', or perhaps your linter is being a PITA?